Exercise_B/index.html

```diff
@@ -4,20 +4,52 @@
     <meta charset="utf-8" />
     <title>Flower Shop BS</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
-
-    <!-- Link and CDN required for bootstrap -->
-
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+    integrity="undefined" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <script src="https://kit.fontawesome.com/78ccc250b3.js" crossorigin="anonymous"></script>
     <!-- Google Font - Lato -->
     <link
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
-    <!-- Custom Stylesheet -->
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
   </head>
 
   <body>
+    <nav class="navbar navbar-expand-lg navbar-dark justify-content-between bg-light p-2 border-bottom">
+      <a class="navbar-brand" href="#">
+        <img alt="Brand" src="assets/images/logo.png">
+      </a>
+    </nav>
+
+      <div class="card bg-dark text-white rounded-0">
+        <img class="card-img" src="./assets/images/flowershop.jpg" alt="Card image">
+        <div class="card-img-overlay d-flex">
+          <div class="align-self-center mx-auto text-center">
+          <h1 class="card-title font-weight-bold">Beautiful Flowers</h1>
+          <a class="btn btn-success btn-lg rounded-pill" href="#" role="button">Learn more</a>
+        </div>
+        </div>
+      </div>

```

Comment:
So in this exercise we are asking you to use the bootstrap column system to put all of the content on one line on an extra large screen
2 icons and their text per line on a medium screen
1 icon and its content per line on a small screen

to achieve this tell the elements what amount of rows they should be allocated using the bootstrap break points.

col-sm-12 col-md-6 col-xl-3
just add that too you elements.